tests/compilers/wast.rs

```diff
@@ -64,6 +64,20 @@ pub fn run_wast(wast_path: &str, compiler: &str) -> anyhow::Result<()> {
     }
     let store = get_store(features, try_nan_canonicalization);
     let mut wast = Wast::new_with_spectest(store);
+    // `bulk-memory-operations/bulk.wast` checks for a message that
+    // specifies which element is uninitialized, but our traps don't
+    // shepherd that information out.
+    wast.allow_trap_message("uninitialized element 2", "uninitialized element");
```

Comment:
What is this? Why this needed always?

Reply:
I have no idea, this is pre-existing, moved out of matches_message_assert_trap() in `tests/lib/wast/src/wast.rs`